src/main/java/org/jenkinsci/plugins/workflow/steps/SynchronousNonBlockingStepExecution.java

```diff
@@ -72,7 +74,14 @@ public void stop(Throwable cause) throws Exception {
 
     @Override
     public void onResume() {
-        getContext().onFailure(new SynchronousResumeNotSupportedException());
```

Comment:
I think you missed two other usages: https://github.com/search?type=code&q=org%3Ajenkinsci+%22new+SynchronousResumeNotSupportedException%28%29%22

Reply:
I noticed in the beginning, but by the I got the code working - I already forgot 😛 

sorry. 

thank you